conf/st2.conf

```diff
@@ -27,6 +27,9 @@ enable = False
 mode = proxy
 logging = st2auth/conf/logging.conf
 
+# Base URL to the API endpoint excluding the version (e.g. http://myhost.net:9101/)
+api_url =
```

Comment:
This needs to point to a URL which is accessible to the clients (usually this will be a public and not an internal / local one) so we can't really know the default value.

We could default to localhost, but this kinda defeats the purpose and will only work if web ui and other clients which use it are running on the same host (and that's something we want to improve with this PR - allowing web ui to run on different host).